data/pathfinder/paizo/player_companion/ranged_tactics_toolbox/rtt_abilities_class.lst

```diff
@@ -55,7 +55,7 @@ Evasion				KEY:Juggler ~ Evasion					CATEGORY:Special Ability	TYPE:ClassFeatures
+Far Strike Monk			KEY:Monk Archetype ~ Far Strike Monk		CATEGORY:Archetype		TYPE:Archetype.MonkArchetype.MonkStunningFist.MonkStillMind.MonkPurityOfBody.MonkDiamondBody																																					PRECLASS:1,Monk=1		PREMULT:1,[PREABILITY:1,CATEGORY=Archetype,Monk Archetype ~ Far Strike Monk],[!PREABILITY:1,CATEGORY=Archetype,TYPE.]																																						DESC:Far strike monks are masters of thrown weapons, from shuriken to throwing axes to spears. The far strike school views thrown attacks as an extension of unarmed strikes, and masters of this school can infuse thrown weapons with their ki.																																																																																																																																					ABILITY:Monk Class Feature|AUTOMATIC|Far Strike Monk ~ Weapon Proficiencies|PRECLASS:1,Monk=1	ABILITY:Monk Class Feature|AUTOMATIC|Far Strike Monk ~ Flurry of Blows|PRECLASS:1,Monk=1	ABILITY:Monk Class Feature|AUTOMATIC|Far Strike Monk ~ Bonus Feats|PRECLASS:1,Monk=1	ABILITY:Monk Class Feature|AUTOMATIC|Far Strike Monk ~ Fast Thrower|PRECLASS:1,Monk=1	ABILITY:Monk Class Feature|AUTOMATIC|Far Strike Monk ~ Invisible Blade|PRECLASS:1,Monk=3	ABILITY:Monk Class Feature|AUTOMATIC|Far Strike Monk ~ Ki Pool|PRECLASS:1,Monk=4	ABILITY:Monk Class Feature|AUTOMATIC|Far Strike Monk ~ Ki Missile|PRECLASS:1,Monk=5	ABILITY:Monk Class Feature|AUTOMATIC|Far Strike Monk ~ Trick Throw|PRECLASS:1,Monk=11																																											COST:0	SOURCEPAGE:p.9	ASPECT:Archetype Base Class|Monk
```

Comment:
PREMULT is not completed... Also, this is not implemented in the new method... I'll accept this, but we should migrate to the newer (Nicer) method.